packages/react-native/ReactCommon/jsi/jsi/decorator.h

```diff
@@ -126,6 +126,9 @@ class RuntimeDecorator : public Base, private jsi::Instrumentation {
       const std::shared_ptr<const PreparedJavaScript>& js) override {
     return plain().evaluatePreparedJavaScript(js);
   }
+  void queueMicrotask(const jsi::Function& callback) override {
+    return plain().queueMicrotask(callback);
+  }
```

Comment:
https://github.com/facebook/hermes/commit/0a632b73b8386dfc9fde0ddca349919c20fe2852 adds this change in JSI. We need it in React Native as well